src/coreclr/ToolBox/superpmi/mcs/verbildump.cpp

```diff
@@ -90,7 +90,12 @@ void DumpSigToConsoleBare(MethodContext* mc, CORINFO_SIG_INFO* pSig)
     for (int i = 0; i < (int)pSig->numArgs; i++)
     {
         DWORDLONG          dl;
+#if defined(TARGET_LOONGARCH64)
```

Comment:
It would be useful to remove all JIT/EE interface changes from this PR so that we can make progress on it independent on https://github.com/dotnet/runtime/pull/62893. Otherwise, the progress on this PR will be blocked by https://github.com/dotnet/runtime/pull/62893.

Reply:
> It would be useful to remove all JIT/EE interface changes from this PR so that we can make progress on it independent on #62893. Otherwise, the progress on this PR will be blocked by #62893.

If I move the JIT/EE interface to #62893, there will be a lot errors within the CI because they are correlative.

Reply:
> It would be useful to remove all JIT/EE interface changes from this PR so that we can make progress on it independent on #62893. Otherwise, the progress on this PR will be blocked by #62893.

If I move the JIT/EE interface to #62893, there will be a lot errors within the CI because they are correlative.

Reply:
> If I move the JIT/EE interface to #62893,

Nothing in VM should depend on JIT/EE interface changes. The only part that depends on JIT/EE interface changes should be the JIT. If you exclude JIT/EE interface changes from this PR, the rest of the VM changes should still build fine.

> I want to merge the #62893 into this PR and close the #62893. 

I think it would be better to keep #62893 separate to make it easier to review. If you merge these two PRs together, I expect that the progress will be slower.

Reply:
OK, I will do it later.